src/Our.Umbraco.NestedContent/Web/UI/App_Plugins/NestedContent/Js/nestedcontent.controllers.js

```diff
@@ -423,6 +426,18 @@ angular.module("umbraco").controller("Our.Umbraco.NestedContent.Controllers.Nest
             }
             return _p8() + _p8(true) + _p8(true) + _p8();
         };
+
+        var getCurrentNodeIndex = function (node) {
+            return node.id === $scope.realCurrentNode.id;
+        }
+
+        var backupGridConfig = function (prop) {
+            return prop.editor == "Umbraco.Grid" ? prop : false;
```

Comment:
That's fair enough, I thought it only seemed right to try and support as many of the native Umbraco property types as possible, and I couldn't see any other way to cater for the grid - Ideally you would just trigger all of the network requests again on open, but I'm not enough of an Angular or Umbraco genius to figure that one out - at least not on company time haha.